commons/Cargo.toml

```
@@ -23,7 +27,12 @@ tempfile = "3"
 thiserror = "1"
 walkdir = "2"
 which_problem = "0.1"
+ascii_table = { version = "4.0.2", features = ["color_codes"] }
+const_format = "0.2.31"
 
 [dev-dependencies]
+indoc = "2.0.1"
 filetime = "0.2"
 toml = "0.7"
+libcnb-test = "0.14.0"
```

The `=` prefix here (used by the other libcnb related deps in this file) makes a difference - otherwise it's equivalent to `^0.14.0`

```suggestion
libcnb-test = "=0.14.0"
```